lib/pharos/host/el7/scripts/reset.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+rm -rf /etc/kubernetes/manifests
+sleep 5
+systemctl stop kubelet
+systemctl disable kubelet
+
+if which docker ; then
```

Comment:
CentOS `/usr/bin/docker` is a wrapper script provided by `docker-common`, not the `docker` package